### DISCUSSION TITLE
SnapBack Baseball Cap

### DISCUSSION THREAD

**Discussion by ernmander** [elementary/store#118] [Released Products]
Maybe an elementary SnapBack Baseball Cap.

**Comment by cassidyjames** [comment]
Printful has a couple of options, but either the embroidery area is small, the bottom of the bill is green, or the quality looks lower than I'd like. Here are some mockups of the nicest looking option in my opinion, though:

![classic-snapback-black-front-60b7f7cb43cd4](https://user-images.githubusercontent.com/611168/120554747-5b1d2400-c3b7-11eb-9d95-22eaeea32ea8.png) | ![classic-snapback-black-silver-front-60b7f7cb43e0b](https://user-images.githubusercontent.com/611168/120554745-58baca00-c3b7-11eb-809d-fdf2cdf4e5ed.png) | ![classic-snapback-maroon-front-60b7f7cb440af](https://user-images.githubusercontent.com/611168/120554726-52c4e900-c3b7-11eb-9a78-af2b2ab07e0a.png) | ![classic-snapback-navy-front-60b7f7cb443e7](https://user-images.githubusercontent.com/611168/120554718-50628f00-c3b7-11eb-80ae-6ffc7a5c89bf.png) | ![classic-snapback-spruce-front-60b7f7cb44700](https://user-images.githubusercontent.com/611168/120554876-8869d200-c3b7-11eb-8dcc-da495c81f912.png) | ![classic-snapback-dark-grey-front-60b7f7cb44a48](https://user-images.githubusercontent.com/611168/120554736-55bfd980-c3b7-11eb-9d0a-0690781c6126.png)
---|---|---|---|---|---
![classic-snapback-black-front-60b7f7cb43992](https://user-images.githubusercontent.com/611168/120554954-9e779280-c3b7-11eb-9919-57dd26c8db4b.png) | ![classic-snapback-black-silver-front-60b7f7cb43f89](https://user-images.githubusercontent.com/611168/120554944-9ae40b80-c3b7-11eb-855c-2c37ce1ea4c9.png) | ![classic-snapback-maroon-front-60b7f7cb4423a](https://user-images.githubusercontent.com/611168/120554900-8f90e000-c3b7-11eb-884c-cba031418a2a.png) | ![classic-snapback-navy-front-60b7f7cb44578](https://user-images.githubusercontent.com/611168/120554887-8b64c280-c3b7-11eb-91a3-0bf59201daed.png) | ![classic-snapback-spruce-front-60b7f7cb448b2](https://user-images.githubusercontent.com/611168/120554700-4b054480-c3b7-11eb-8301-66c4e0b10ff7.png) | ![classic-snapback-dark-grey-front-60b7f7cb44c13](https://user-images.githubusercontent.com/611168/120554928-94ee2a80-c3b7-11eb-8f72-d184f2654ea2.png)


The one thing I don't like is the bottom of the bill:

![classic-snapback-black-product-details-60b7f979949d0](https://user-images.githubusercontent.com/611168/120555357-31183180-c3b8-11eb-9fc5-e79643baeee2.png)

**Reply by ernmander** [reply]
They look quite nice. Do they do a trucker hat type, you know the baseball cap with a mesh back? Might be worth playing around with trying the logo to left or right of centre, see if that does anything to the overall look. They do look nice though.

**Reply by ernmander** [reply]
Also I agree with you on the bottom of the bill

**Reply by danirabbit** [reply]
I think for an embroidery that using the inverted version of the logo makes more sense than something with a large fill area like this. That would also help with sizing

**Reply by ernmander** [reply]
Absolutely agree. Embroidered logo is so much better than printed. Good luck with the elementary developer workshop later this month.

**Comment by cassidyjames** [comment]
Here's another attempt with the inverted/thin logo.

![Screenshot from 2021-07-08 12 44 46](https://user-images.githubusercontent.com/611168/124974702-579a3f80-dfea-11eb-9c2c-aaa987ede5ef.png)

**Reply by ernmander** [reply]
That looks a lot better.

**Reply by danirabbit** [reply]
Wow is that as big as it can be? It looks so small

**Reply by ernmander** [reply]
As I keep saying try to print the logo off centre, why should it be in the centre of the cap. Try the logo to the left or right. I got the size of the logo just right in my mock up ;) 

Guys if you don't want a baseball/trucker cap just say.

**Reply by cassidyjames** [reply]
@danrabbit yep, that's as big of an embroidery area as they offer on this hat. I can check the other hats again and see if the area differs or not.

@ernmander I played with it on this model and there's just not much room to play with, and it looks weird off-center unless you go substantially smaller, but then you can only get so small with the thin embroidery strokes.

**Reply by ernmander** [reply]
Thank you for the heads up and trying this.

**Comment by btkostner** [comment]
This has been recently added to the store https://store.elementary.io/products/235661638!

**Comment by ernmander** [comment]
Great to see this in the Store. Really enjoyed the process of taking to you guys.
